test/e2e/common/storage/configmap_volume.go

```diff
@@ -556,17 +556,19 @@ var _ = SIGDescribe("ConfigMap", func() {
 	// Slow (~5 mins)
 	ginkgo.It("Should fail non-optional pod creation due to configMap object does not exist [Slow]", func(ctx context.Context) {
 		volumeMountPath := "/etc/configmap-volumes"
-		pod, err := createNonOptionalConfigMapPod(ctx, f, volumeMountPath)
-		framework.ExpectError(err, "created pod %q with non-optional configMap in namespace %q", pod.Name, f.Namespace.Name)
+		pod := createNonOptionalConfigMapPod(ctx, f, volumeMountPath)
+		getPod := e2epod.Get(f.ClientSet, pod)
+		gomega.Consistently(ctx, getPod).WithTimeout(framework.NewTimeoutContextWithDefaults().PodStart).Should(e2epod.BeInPhase(v1.PodPending))
```

Comment:
PodStart = 5min is what was used before.

Note that https://github.com/kubernetes/kubernetes/pull/114783 will make it simpler to access that timeout.

Reply:
/hold

Let's use the timeouts from that PR.

Reply:
/hold cancel

This is done.